lib/elixometer.ex

```diff
@@ -348,8 +348,7 @@ defmodule Elixometer do
       subscribe_options = cfg[:subscribe_options] || []
 
       if reporter do
-        :exometer.info(metric_name)
-        |> Keyword.get(:datapoints)
+        :exometer.info(metric_name, :datapoints)
```

Comment:
What changed? I looked at head, and Ulf still has two versions of this function. 
Can you write a test case?